### PR TITLE
Initial the basic command framework

### PR DESCRIPTION
**Pull request by 0xff-dev**
Initial the basic command framework

### CODE REVIEW

**Comment by bjwswang** [comment]
add `.github` into this project

File: hack/lib/golang.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+function go::verify_go_version() {
+	if [[ -z "$(command -v go)" ]]; then
+		echo "go is not installed"
+		exit 1
+	fi
+
+	local go_version
+	IFS=" " read -ra go_version <<<"$(GOFLAGS='' go version)"
+	local minimum_go_version
+	minimum_go_version=go1.18.3
```
Comment (Abirdcfly) [inline hack/lib/golang.sh:70]: when go.mod set go version is 1.20, minimum_go_version should also update to 1.20

File: .github/workflows/image-build.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        if: github.event_name == 'push'
+        uses: docker/login-action@v2
+        with:
+          username: hyperledgerk8s
+          password: ${{ secrets.DOCKER_TOKEN }}
+      - name: retag and push
+        if: github.event_name == 'push'
+        run: |
+          image=$(docker images --format '{{.Repository}}:{{.Tag}}' | grep bc-saas)
+          docker tag ${image} hyperledgerk8s/bc-saas:latest
```
Comment (Abirdcfly) [inline .github/workflows/image-build.yaml:39]: also saas -> cli

File: CODE_OF_CONDECT.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+Bestchains/saas follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (Abirdcfly) [inline CODE_OF_CONDECT.md:3]: saas -> cli

File: .github/workflows/unit-tests.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+env:
+  GO_VER: 1.18
```
Comment (bjwswang) [inline .github/workflows/unit-tests.yaml:14]: since we use go1.20 in go modules , should change this to 1.20 too.

**Comment by bjwswang** [comment]
As discussed yester, we define commands under `cmd/cli` and implement under different packages under `pkg/depository` `pkg/organization` ???

**Review by bjwswang — APPROVED** [review]
/lgtm

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**